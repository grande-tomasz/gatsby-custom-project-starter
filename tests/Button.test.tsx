import styled from 'styled-components';
import React from 'react';
import renderer from 'react-test-renderer';

const Button = styled.button`
  color: red;
`;

test('Button matches snapshot', () => {
  const tree = renderer.create(<Button />).toJSON();
  expect(tree).toMatchSnapshot();
});
