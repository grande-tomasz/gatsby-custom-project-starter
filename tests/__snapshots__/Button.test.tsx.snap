// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button matches snapshot 1`] = `
.c0 {
  color: red;
}

<button
  className="c0"
/>
`;
